app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class AdminController < ApplicationController
+  def index; end
+end
```

Comment:
If you'll look at the following tasks, you'll see that you have to work with products resources, so you don't need admin controller here, as we don't have any admin model or smth like.